### PR TITLE
Fspes 50

### PR DESCRIPTION
**Pull request by MatteoDelOmbra**
# *Task Update PR template*

## Review Checklist

- [ ] Task version updated (x.x.0)
- [ ] CHANGELOG.md updated
- [ ] Solution builds
- [ ] Warnings resolved (if possible)
- [ ] Typos resolved
- [ ] Tests cover new code
- [ ] Description how to run tests locally added to README.md (if needed)
- [ ] All tests pass locally

## Summary by CodeRabbit

* **New Features**
  * Option to control handling of illegal XML node names (error or sanitize) with a customizable prefix.

* **Changed**
  * Parse task now returns parsed data only; XML and JSON outputs removed.
  * Default header delimiter and header-whitespace defaults updated.

* **Tests**
  * Unit tests added/updated to cover illegal-node handling and adjusted parsing behavior.

* **Chores**
  * Project versions and changelogs updated.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: Frends.CSV.Parse/Frends.CSV.Parse/Parse.cs
Comment (MichalFrends1) [inline Frends.CSV.Parse/Frends.CSV.Parse/Parse.cs]: remove Jtoken and Xml from result docs

File: Frends.CSV.ConvertToXML/Frends.CSV.ConvertToXML/Helpers/XmlHandler.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        
+        // first character
+        var first = nodeName[0];
+
+        if (!(char.IsLetter(first) || first == '_'))
+        {
+            if (illegalNodeNameAction == IllegalNodeNameAction.ThrowError)
+                throw new Exception($"Illegal first character '{first}' in nodeName \"{nodeName}\"");
+
+            nodeName = $"{nodeNamePrefix}{nodeName}";
```
[the remark this answers is not in the document]
Comment (MichalFrends1) [reply Frends.CSV.ConvertToXML/Frends.CSV.ConvertToXML/Helpers/XmlHandler.cs:39]: Please add a test covering empty headers.

**Review by MichalFrends1 — CHANGES_REQUESTED** [review]

**Review by MichalFrends1 — APPROVED** [review]
